### ISSUE TITLE
REDESIGN THE USER PROFILE PAGE

### ISSUE THREAD

**Issue by RATED-R-SUNDRAM** [RATED-R-SUNDRAM/LINKED_OUT#1]
Go to http://127.0.0.1:8000/profile/<user_name> add some custom styles forex. redesign the layout, add borders, add some attractive color combinations, etc. Any other change to enhance the overall look of this page without disturbing the responsivity (bootstrap suggested) is encouraged.

**Comment by artimanputra** [comment]
Can I work On this issue, Could you please assign it to me?

**Comment by RATED-R-SUNDRAM** [comment]
Hey, This issue has been assigned to you, it's a very big issue, keep in check with me, make regular commits as we have not finalized how the final draft will look, you are free to go ahead and contribute in your way for reference I have made a ppt "https://docs.google.com/presentation/d/1zGwg4G-D1gtarR7S9HBHGUBAfGOovcen/edit?usp=sharing&ouid=109332390191761610544&rtpof=true&sd=true' again this is not exactly how I want it to be, go ahead and keep on adding intermittent pull requests.

**Comment by RATED-R-SUNDRAM** [comment]
Also please open more issues you find useful to be done in context with the front-end part.

**Comment by artimanputra** [comment]
> Hey, This issue has been assigned to you, it's a very big issue, keep in check with me, make regular commits as we have not finalized how the final draft will look, you are free to go ahead and contribute in your way for reference I have made a ppt "https://docs.google.com/presentation/d/1zGwg4G-D1gtarR7S9HBHGUBAfGOovcen/edit?usp=sharing&ouid=109332390191761610544&rtpof=true&sd=true' again this is not exactly how I want it to be, go ahead and keep on adding intermittent pull requests.

Sure, If I need any help where can I talk with you?

**Comment by artimanputra** [comment]
> Also please open more issues you find useful to be done in context with the front-end part.

We can open an issue for the footer part, shall I create it for other?

**Comment by RATED-R-SUNDRAM** [comment]
Yeah sure go ahead, write a suitable heading and appropriate details of the issue and open it, Leave the labels and assignees to me if no one responds, I might assign it to you in a couple of days.

**Comment by artimanputra** [comment]
sure

**Comment by RATED-R-SUNDRAM** [comment]
@artimanputra since you are working on a bigger issue, make sure you make frequent PRs and git pulls.

**Comment by artimanputra** [comment]
> @artimanputra since you are working on a bigger issue, make sure you make frequent PRs and git pulls.

okay
